src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -87,46 +85,33 @@ Git: 0.7-summer-2025-ncr-444-g89b9baa9b-dirty
         </mandatoryConform>
       </otherwiseConform>
     </attribute>
-    <attribute side="server" code="0x06" name="TCMinRequiredVersion" define="TC_MIN_REQUIRED_VERSION" type="int16u" apiMaturity="provisional">
+    <attribute side="server" code="0x06" name="TCMinRequiredVersion" define="TC_MIN_REQUIRED_VERSION" type="int16u">
       <access op="read" privilege="administer"/>
-      <otherwiseConform>
-        <provisionalConform/>
-        <mandatoryConform>
-          <feature name="TC"/>
-        </mandatoryConform>
-      </otherwiseConform>
+      <mandatoryConform>
+        <feature name="TC"/>
+      </mandatoryConform>
     </attribute>
-    <attribute side="server" code="0x07" name="TCAcknowledgements" define="TC_ACKNOWLEDGEMENTS" type="bitmap16" apiMaturity="provisional">
+    <attribute side="server" code="0x07" name="TCAcknowledgements" define="TC_ACKNOWLEDGEMENTS" type="bitmap16">
       <access op="read" privilege="administer"/>
-      <otherwiseConform>
-        <provisionalConform/>
-        <mandatoryConform>
-          <feature name="TC"/>
-        </mandatoryConform>
-      </otherwiseConform>
+      <mandatoryConform>
+        <feature name="TC"/>
+      </mandatoryConform>
     </attribute>
-    <attribute side="server" code="0x08" name="TCAcknowledgementsRequired" define="TC_ACKNOWLEDGEMENTS_REQUIRED" type="boolean" apiMaturity="provisional" default="1">
+    <attribute side="server" code="0x08" name="TCAcknowledgementsRequired" define="TC_ACKNOWLEDGEMENTS_REQUIRED" type="boolean" default="1">
       <access op="read" privilege="administer"/>
-      <otherwiseConform>
-        <provisionalConform/>
         <mandatoryConform>
           <feature name="TC"/>
         </mandatoryConform>
-      </otherwiseConform>
     </attribute>
-    <attribute side="server" code="0x09" name="TCUpdateDeadline" define="TC_UPDATE_DEADLINE" type="int32u" isNullable="true" apiMaturity="provisional">
+    <attribute side="server" code="0x09" name="TCUpdateDeadline" define="TC_UPDATE_DEADLINE" type="int32u" isNullable="true">
       <access op="read" privilege="administer"/>
-      <otherwiseConform>
-        <provisionalConform/>
-        <mandatoryConform>
-          <feature name="TC"/>
-        </mandatoryConform>
-      </otherwiseConform>
+      <mandatoryConform>
+        <feature name="TC"/>
+      </mandatoryConform>
     </attribute>
     <attribute code="0x000A" side="server" name="RecoveryIdentifier" define="RECOVERY_IDENTIFIER" type="octet_string" length="8" minLength="8" default="0" apiMaturity="provisional">
       <access op="read" privilege="manage"/>
       <otherwiseConform>
-        <provisionalConform/>
```

Comment:
RecoveryIdentifier should still be marked provisional, it is still marked so in the spec

Reply:
will update - I think I manually did some updates and then backed off a few again ... and that apparently was incomplete.

Zap currently does not actually use this tag, however it should be there